app/assets/javascripts/components/form_buttons.js

```diff
@@ -0,0 +1,8 @@
+ManageIQ.angular.app.directive('formButtons', function() {
+  return {
+    restrict: 'E',
+    scope: false,
+    controller: 'buttonGroupController',
+    templateUrl: 'buttons.html'
```

Comment:
I assume this is provided by `angular-rails-templates`, but still, it should have a more descriptive name than `buttons.html`, ideally something like `components/form_buttons.html` or even `components/form_buttons/something.html`..